common/test/acceptance/pages/studio/settings.py

```diff
@@ -17,25 +17,45 @@ class SettingsPage(CoursePage):
     def is_browser_on_page(self):
         return self.q(css='body.view-settings').present
 
+    def get_elements(self, css_selector):
+        self.wait_for_element_presence(
+            css_selector,
+            'Elements matching "{}" selector are present'.format(css_selector)
+        )
+        results = self.q(css=css_selector)
+        return results
+
+    def get_element(self, css_selector):
+        results = self.get_elements(css_selector=css_selector)
+        return results[0] if results else None
```

Comment:
Ah, true. But your other code is implicitly relying on it to. For example entrance_exam_field returns self.get_element('#entrance-exam-enabled'). If that was None then this code from the require_entrance_exam method would fail.
```
    checkbox = self.entrance_exam_field
    selected = checkbox.is_selected()
```
Which in itself is a good reminder that the docstrings for all the new methods that return values should explicitly say that they return foo or ... empty list, None, whatever else they might sometimes return ... and when you expect them to raise an error (e.g. when the element is not found).